app/jobs/mentions/create_all_job.rb

```diff
@@ -0,0 +1,10 @@
+module Mentions
+  class CreateAllJob < ApplicationJob
+    queue_as :mentions_create_all
+
+    def perform(notifiable_id, notifiable_type)
+      notifiable = notifiable_type.constantize.find_by(id: notifiable_id)
```

Comment:
This potential raises an error though (totally fine if that's what we want). Though very unlikely that it's going to happen, If I run `"Acme".constantize.find_by(id: 1)` I get:

```shell
> "Acme".constantize.find_by(id: 1)
NameError: uninitialized constant Acme
```

Reply:
I suppose that having `uninitialized constant` is ok because it's an exceptional situation and we would like to know about that. But if a class name that exists but not expected is passed, e.g. `User` the following behavior is not that predictable and can cause bugs. So I suggest checking and being confident about what's happening.